src/containers/AuthorsList.js

```diff
@@ -66,30 +86,43 @@ class AuthorsList extends React.Component {
       }
       return authorItemObject
     }
-    const authorsArray = _.map(authorsInList, iteratee)
+    const authorsArray = _.map(authorsIdList, iteratee)
 
-    // Callback for sensor is triggered to seen
+    // Callback for sensor being seen
     let handleSeen = (isVisible) => {
-      if (currentPage>REQUEST_PAGE_START_FROM && isVisible === true) {
-        fetchNextPageAuthors()
+      if (currentPage>NUMBER_OF_FIRST_RESPONSE_PAGE && isVisible === true) {
+        return searchAuthorsIfNeeded('')
+      }
+    }
+    // Callback for loadmore button being clicked
+    let handleClickLoadmore = () => {
+      return searchAuthorsIfNeeded('')
+    }
+    
+    let changeListTo = (listType) => {
+      if (listType === 'authorsList' || listType === 'searchedAuthorsList') {
+        this.setState({ whichAuthorsListToRender: listType })
```

Comment:
is it possible to have another value rather than `authorsList ` and `searchedAuthorsList ` ?